### PR TITLE
Add additional Google Analytics cookies to the cookie deny list

### PR DESCRIPTION
**Pull request by Mynyx**
See https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage

### CODE REVIEW

**Review by leofeyer — CHANGES_REQUESTED** [review]
We do not need two separate sections for Google Analytics. Just keep the original comment as it was and add the missing names. (We might even be able to use a regex such as `'__utm[abctvz]'` instead of multiple lines).

**Comment by Toflar** [comment]
I would maybe just strip `__utm*`. I can't think of any developer that would prefix their cookies with this string and use it internally :D

**Comment by Mynyx** [comment]
> We do not need two separate sections for Google Analytics. Just keep the original comment as it was and add the missing names. (We might even be able to use a regex such as `'__utm[abctvz]'` instead of multiple lines).

Okay, I have additionally added ``_dc_gtm_<property-id>``.

> ``_gat``: Used to throttle request rate. If Google Analytics is deployed via Google Tag Manager, this cookie will be named ``_dc_gtm_<property-id>``.

File: core-bundle/src/EventListener/HttpCache/StripCookiesSubscriber.php [hunk 1]
```diff
@@ -30,8 +30,10 @@ class StripCookiesSubscriber implements EventSubscriberInterface
         '_ga',
         '_gid',
         '_gat',
+        '_dc_gtm_.+',
         'AMP_TOKEN',
         '_gac_.+',
+        '__utm[atbczv]',
```
Comment (leofeyer) [inline core-bundle/src/EventListener/HttpCache/StripCookiesSubscriber.php]: According to https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage#experiments, there is also `__utmx` and `__utmxx`, therefore we should probably use `'__utm.+'` as @Toflar has suggested.

**Review by leofeyer — CHANGES_REQUESTED** [review]

**Comment by Mynyx** [comment]
> I would maybe just strip `__utm*`. I can't think of any developer that would prefix their cookies with this string and use it internally :D

That was my first implementation, but I thought I should by more careful. 😄

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @Mynyx.

**Pull request merged**